### PR TITLE
:arrow_up: rowan

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Comment by matklad** [comment]
bors r+
🤖

File: Cargo.toml [hunk 1]
```diff
@@ -11,3 +11,4 @@ incremental = true
 debug = 0 # set this to 1 or 2 to get more useful backtraces in debugger
 
 [patch.'crates-io']
+# rowan = { path = "../rowan" }
```
Comment (mikeyhew) [inline Cargo.toml]: should this line be removed?

Comment (matklad) [reply Cargo.toml]: Nope: I always forget the patchin syntax, and this helps

Comment (mikeyhew) [reply Cargo.toml]: ah, understood

**Pull request merged**